doc/classes/NavigationServer2D.xml

```diff
@@ -344,6 +344,14 @@
 				Returns the enter cost of this [param link].
 			</description>
 		</method>
+		<method name="link_get_iteration_id" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="link" type="RID" />
+			<description>
+				Returns the current iteration ID of the navigation link. Every time the navigation link changes and synchronizes, the iteration ID increases. An iteration ID of [code]0[/code] means the navigation link has never synchronized.
```

Comment:
Nitpick
```suggestion
				Returns the current iteration ID of the given navigation link. Every time the navigation link changes and synchronizes, the iteration ID increases. An iteration ID of [code]0[/code] means the navigation link has never synchronized.
```
or
```suggestion
				Returns the current iteration ID of the given navigation [param link]. Every time the navigation link changes and synchronizes, the iteration ID increases. An iteration ID of [code]0[/code] means the navigation link has never synchronized.
```

Reply:
It is the same wording as for the similar iteration id region and map functions. You might want to make a documentation PR that updates them all at the same time.